### ISSUE TITLE
Validation loss is not stable for both classification and regression 

### ISSUE THREAD

**Issue by alsheabi** [zylo117/Yet-Another-EfficientDet-Pytorch#707]
Any suggestion or idea about this issue?

![loss](https://user-images.githubusercontent.com/6864735/146680718-0c02ee21-c9b1-4218-9a81-e17efc089a2b.png)

**Comment by zylo117** [comment]
category id starts from 1, 0 is the background

**Comment by alsheabi** [comment]
> category id starts from 1, 0 is the background

@zylo117 Thank you for replying, regards category id start from 1 I did that for coco, but I don't get 
what do you mean about 0 for background :( do you mean segmentation in Annotation? Could you please tell me what it's mean?

![cate](https://user-images.githubusercontent.com/6864735/146935755-69ea2149-1c6d-4ee5-abaf-d0674110f6c6.png)
 and Annotation for Bbox looks like this:
![ann](https://user-images.githubusercontent.com/6864735/146936855-6ebd5cee-272f-4cbd-aad7-feb56d417ca4.png)
 Or you mean IoU it's already 0 

![back](https://user-images.githubusercontent.com/6864735/146941922-c22f6675-fd1a-4a0a-b16c-40279378f6f1.png)
